tests/integration/relations/test_db_admin.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+import ast
+import json
+import logging
+from pathlib import Path
+
+import pytest
+import yaml
+from landscape_api.base import run_query
+from pytest_operator.plugin import OpsTest
+
+from tests.integration.helpers.helpers import (
+    deploy_and_relate_bundle_with_pgbouncer_bundle,
+    deploy_postgres_bundle,
+    get_backend_user_pass,
+    get_legacy_relation_username,
+)
+from tests.integration.helpers.postgresql_helpers import (
+    check_database_users_existence,
+    check_databases_creation,
+)
+
+logger = logging.getLogger(__name__)
+
+HAPROXY_APP_NAME = "haproxy"
+LANDSCAPE_APP_NAME = "landscape-server"
+LANDSCAPE_SCALABLE_BUNDLE_NAME = "ch:landscape-scalable"
+RABBITMQ_APP_NAME = "rabbitmq-server"
+DATABASE_UNITS = 3
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+PGB = METADATA["name"]
+
+
+@pytest.mark.dev
+@pytest.mark.legacy_relation
+async def test_landscape_scalable_bundle(ops_test: OpsTest) -> None:
+    """Deploy Landscape Scalable Bundle to test the 'db-admin' relation."""
+    config = {
+        "extra-packages": "python-apt postgresql-contrib postgresql-.*-debversion postgresql-plpython.*"
+    }
+    backend_relation = await deploy_postgres_bundle(
+        ops_test, pg_config=config, db_units=DATABASE_UNITS
+    )
+    await ops_test.model.applications[PGB].set_config({"listen_port": "5432"})
+
+    async with ops_test.fast_forward():
+        # Deploy and test the Landscape Scalable bundle (using this charm).
+        relation_id = await deploy_and_relate_bundle_with_pgbouncer_bundle(
+            ops_test, LANDSCAPE_SCALABLE_BUNDLE_NAME, LANDSCAPE_APP_NAME, relation_name="db-admin"
+        )
+    await check_databases_creation(
+        ops_test,
+        [
+            "landscape-account-1",
+            "landscape-knowledge",
+            "landscape-main",
+            "landscape-package",
+            "landscape-resource-1",
+            "landscape-session",
+        ],
+    )
+
+    landscape_users = get_legacy_relation_username(ops_test, relation_id)
+    pgb_user, pgb_pass = await get_backend_user_pass(ops_test, backend_relation)
+
+    await check_database_users_existence(ops_test, landscape_users, [], pgb_user, pgb_pass)
+
+    # Configure and admin user in Landscape and get its API credentials.
```

Comment:
Probably the same type exists on PostgreSQL integration test.
```suggestion
    # Configure an admin user in Landscape and get its API credentials.
```